TaskOne/hash.js

```diff
@@ -0,0 +1,25 @@
+var crypto = require("crypto");
+var shaAlgorithm = "sha1";
+var mdAlgorithm = "md5";
+const fs = require("fs");
+function generateHash(filename) {
+  s = fs.ReadStream(filename);
+  var shasum = crypto.createHash(shaAlgorithm);
+  var mdsum = crypto.createHash(mdAlgorithm);
+  s.on("data", function (data) {
+    shasum.update(data);
+    mdsum.update(data);
+  });
+  s.on("end", function () {
+    var shash = shasum.digest("hex");
+    var mhash = mdsum.digest("hex");
+    var buff = filename + "  " + shash + "  " + mhash + "\n";
+    fs.appendFile("write.txt", buff, function (err) {
+      if (err) throw err;
+      console.log(filename + " hash saved!");
+    });
```

Comment:
as an improvement, take a callback in generateHash method and call the callback with two hash keys..
and outside you can handle appendfile logic.

we should always try to have a single responsibility for a function. A good practice.

you definition in line 5 would be `function generateHash(filename, callback) {`
and in line 17 you will call it `callback(sha1, md5)`

in the other file where you called this you write the `appendFile` logic.